content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -17,7 +18,7 @@ const ActivityFeedItem = React.createClass({
           <a className="feed-link" href={site.url} ref="link">{prettyUrl(site.url)}</a>
         </div>
         <div className="feed-stats">
-          <div>1:26pm</div>
+          <div>{site.lastVisitDate && moment(site.lastVisitDate / 1000).format("h:mma")}</div>
```

Comment:
**Heads up:** There was talk in [#186](https://github.com/mozilla/activity-streams/pull/186/files#r54038788) of dividing `lastVisitDate` by 1000 in lib/PlacesProvider.js SQL queries.